README.md

```diff
@@ -129,19 +130,32 @@ sign-offs to merge PRs (and for large or contentious issues we may
 wait for more). For typos or other small fixes to documentation we
 relax this to a single sign-off.
 
-### Related Projects
+### The cats ecosystem
 
-There are many projects that integrate with Cats:
+#### Generic libraries to support pure functional programming
 
- * [Circe](https://github.com/circe/circe): pure functional JSON library.
+ * [cats-effects](https://github.com/typelevel/cats-effect): provide a standard `IO` type for the cats ecosystem
```

Comment:
Also, this project doesn't provide just `IO`, but also the `Sync`, `Async` and `Effect` type classes, which I think are important, since this is an innovation.